tutordiscovery/templates/discovery/build/discovery/Dockerfile

```diff
@@ -5,8 +5,12 @@ RUN apt update && \
   build-essential libffi-dev libmysqlclient-dev libxml2-dev libxslt-dev libjpeg-dev libssl-dev
 ENV LC_ALL en_US.UTF-8
 
+ARG DISCOVERY_REPOSITORY=https://github.com/edx/course-discovery.git
+ARG DISCOVERY_VERSION=open-release/ironwood.2
+ARG DISCOVERY_VERSION_DATE=20200227
 RUN mkdir -p /openedx/discovery && \
-    git clone https://github.com/edx/course-discovery.git --branch open-release/ironwood.2 --depth 1 /openedx/discovery
+    echo "Pulling $DISCOVERY_VERSION tag from $DISCOVERY_REPOSITORY ($DISCOVERY_VERSION_DATE)" && \
```

Comment:
You can also get rid of this "echo" statement -- or not, it's how you prefer.

Reply:
Let me remove it.